### ISSUE TITLE
API to expose to AAs which identity attributes businesses want to accept

### ISSUE THREAD

**Issue by rrix** [consumer-reports-digital-lab/data-rights-protocol#52]
One piece of really salient feedback we heard at DEFCON was that businesses don't **need** or even **want** to accept all identity attributes. I've been thinking about this for a while and it's time to work through this.

It seems simple off-hand to extend `data-rights.json` to include a list of claims:

    {
      "version": "0.6",
      "actions": ["sale:opt-out", "sale:opt-in", "access", "deletion"],
      "user-attributes": ["name", "email"]  
    }

But CCPA for example has different verification requirements for data sale opt-out requests compared to deletion or access. So maybe:

    {
        "version": "0.6",
        "actions": {
            "sale:opt-out": ["email", "email_verified"],
            "sale:opt-in": ["email", "email_verified"],
            "access": ["name", "email", "email_verified",
                       "phone_number", "phone_number_verified",
                       "power_of_attorney"],
            "deletion": ["name", "email", "email_verified",
                         "phone_number", "phone_number_verified",
                         "power_of_attorney"]
        }
    }

But this is verbose, while exposing yet more nuance

-   should the business be able to signal **required** attributes? ("we accept POA but require verified email") or is this over-optimizing?
-   what about custom claims?
-   should this new schema have a new field name, if so what?

**Comment by kevinr** [comment]
> should `GET .well-known/data-rights.json` be expected/asked to respect `Accept-Language` header to localize these responses?

No; Accept-Language is semantically about what natural language should be returned *in the response*, not about what legal jurisdiction the requester is in or asking about, and should return the same thing regardless, just translated.
